core/domain/exp_services.py

```diff
@@ -352,6 +353,8 @@ def apply_change_list(exploration_id, change_list):
         exploration_id: str. The id of the exploration to which the change list
             is to be applied.
         change_list: list(ExplorationChange). The list of changes to apply.
+        frontend_version: int. Version of the exploration on which the user
+            was working.
```

Comment:
Would "The exploration version number on the client side." make sense here?

Reply:
Changed it.